lib/rubocop/cop/style/empty_heredoc.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for using empty heredoc to reduce redundancy.
+      #
+      # @example
+      #
+      #   # bad
+      #   <<~EOS
+      #   EOS
+      #
+      #   <<-EOS
+      #   EOS
+      #
+      #   <<EOS
+      #   EOS
+      #
+      #   # good
+      #   ''
+      #
+      #   # bad
+      #   do_something(<<~EOS)
+      #   EOS
+      #
+      #   do_something(<<-EOS)
+      #   EOS
+      #
+      #   do_something(<<EOS)
+      #   EOS
+      #
+      #   # good
+      #   do_something('')
+      #
+      class EmptyHeredoc < Base
+        include Heredoc
+        include RangeHelp
+        extend AutoCorrector
+
+        MSG = "Use `''` instead of heredoc."
```

Comment:
I guess that will be tied to whether the user selected single or double quoted string literals in their config. You may cheap by saying "an empty string literal".

Reply:
Sure! I've updated!

Reply:
Similarly, the autocorrector says `corrector.replace(node, "''")`. Though maybe the autocorrector will see the `''` and swap it for `""` because it just inserted an invalid rule. I'm not sure how the autocorrector works. Just, the code looks like it'll have the same problem as the `MSG` did.